tech.json

```
@@ -495,12 +500,12 @@
     },
     {
       "name": "Bomb Jumps",
-      "description": "Techs that have to do with using bomb explosions to propel Samus",
+      "description": "Techs that have to do with using bomb explosions to propel Samus.",
       "techs": [
         {
           "name": "canUnmorphBombBoost",
           "requires": [ "h_canBombThings" ],
-          "note": "A tech that involves mid-air morphing to drop a Morph Bomb or PB, then mid-air unmorphing to briefly hover above the bomb, then using the bomb blast to go just a bit higher than max jump height"
+          "note": "A tech that involves mid-air morphing to place a Morph Bomb or Power Bomb, then mid-air unmorphing to briefly hover above the bomb, in order to use the bomb blast to go just a bit higher than max jump height."
```

```suggestion
          "note": "A tech that involves mid-air morphing to place a bomb or Power Bomb, then mid-air unmorphing to briefly hover above the bomb, in order to use the bomb blast to go just a bit higher than max jump height."
```
I think it's generally better to just say "bomb". According to the wiki, the term "Morph Bomb" mainly comes from SMZ3 to distinguish it from ALTTP bombs (https://wiki.supermetroid.run/Bomb), which isn't necessary in a pure SM context.